### PR TITLE
Bug 571779 - Inverted selection behaviour of SwitchButton

### PR DESCRIPTION
**Pull request by lcaron**

### CODE REVIEW

File: widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java [hunk 1]
```diff
@@ -161,38 +155,33 @@
...
 		super(parent, style | SWT.DOUBLE_BUFFERED);
 
-		selection = false;
+		selection = true;
 		text = "";
 		textForSelect = "On";
 		textForUnselect = "Off";
 		round = true;
 		borderColor = null;
 		focusColor = getDisplay().getSystemColor(SWT.COLOR_YELLOW);
```
Comment (laeubi) [inline widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java:164]: I found this yellow really... disturbing... is this by design? Maybe one of the COLOR_WIDGET_* constants would be more suitable?

File: widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java [hunk 2]
```diff
@@ -161,38 +155,33 @@
 	public SwitchButton(final Composite parent, final int style) {
 		super(parent, style | SWT.DOUBLE_BUFFERED);
 
-		selection = false;
+		selection = true;
```
Comment (laeubi) [inline widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java:158]: I thing "false" as a default is more like it is with standard swt widget, e.g. a checkbox is initially unselected, a toogle button is not toggled and so on...

File: widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java (hunk 1 above)
Comment (lcaron) [reply widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java:164]: When I worked on this PR, I noticed this is weird. The next commit will fix it

File: widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java (hunk 2 above)
Comment (lcaron) [reply widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java:158]: Good suggestion, it is solved in the next commit

**Pull request merged**